SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialEventTracer.cpp

```diff
@@ -0,0 +1,322 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "Interop/Connection/SpatialEventTracer.h"
+
+#include "UObject/Class.h"
+#include "GameFramework/Actor.h"
+
+#include <WorkerSDK/improbable/c_trace.h>
+
+DEFINE_LOG_CATEGORY(LogSpatialEventTracer);
+
+using namespace SpatialGDK;
+using namespace worker::c;
+
+void MyTraceCallback(void* UserData, const Trace_Item* Item)
+{
+	switch (Item->item_type)
```

Comment:
It could be worth having filter modes for these streams, perhaps a `message type` based subscription so that the event trace can pick up the events its interested in but other events can exist also

Reply:
This isn't in the scope of the task right now though but it may require refactoring later on